registry.adoc

```diff
@@ -1392,14 +1392,17 @@ implemented against.
   * attr:type - required if the attr:supported attribute is not
     `'disabled'`.
     Must be either `'device'` or `'instance'`, if present.
-  * attr:depends - optional.
+  * [[depends-expressions]] attr:depends - optional.
     String containing a boolean expression of one or more API core version
     and extension names.
     The extension requires the expression in the string to be satisfied to
     use any functionality it defines (for instance extensions), or to use
     any device-level functionality it provides (for device extensions).
     Supported operators include `,` for logical OR, `+` for logical AND, and
     `(` `)` for grouping.
+    `,` and `+` are of equal precedence, and lower than `(` `)`.
+    Expressions must be evaluated left-to-right for operators of the same
+    precedence.
```

Comment:
@oddhack so `a+b,c` is `(a+b),c` and `a,b+c` is `(a,b)+c`?

I'm not a fan of this (not intuitive for me, which makes it error-prone for anyone else who might think like me), but at least now it's documented, so thank you :+1: